### ISSUE TITLE
Is it possible to optimize `style.text` to be like `style_text`?

### ISSUE THREAD

**Issue by mokeyish** [ryansolid/dom-expressions#251]
When using tailwindcss/unocss, there are places where styles are duplicated, so this way is needed to reference duplicated styles.

Is it possible to optimize `style.text` to be like `style_text`?

![](https://media.discordapp.net/attachments/722131463889223772/1107631318449258596/c1ef69642c76b145.png?width=1920&height=525)

The reproduce code:

https://playground.solidjs.com/anonymous/4d3dfcf1-136f-4a7c-ba38-1cdfa8418e9f

**Comment by ryansolid** [comment]
Not easily.. or rather it would require resolution. I suppose we could try but keep in mind `.` heuristics matter for possible reactive wrapping since getters/proxies. Even if we could optimize this case where everything is inlined like that it would drop back as soon as it was in another file.

**Comment by mokeyish** [comment]
Thanks for your explaination. Maybe it needs something like `/*@once*/ ` to mark it non-reactive.🤣

**Comment by intrnl** [comment]
I'm rather wary on objects even if they are defined as a constant, since they are still mutable, *however*, namespace imports should be generally fine if you're willing to author these classes on a separate file as a tradeoff.

re: #283 on the matter.

**Comment by intrnl** [comment]
Still not inline-able in the template, but #283 should at least address not having to wrap namespace imports under effects, the only tradeoff being that you'd have to author these classes in a separate file.

**Comment by ryansolid** [comment]
yeah I consider the merged PR as far as we can take this where things are at.
